### PR TITLE
Categories section of quiz

### PR DESCRIPTION
**Pull request by utkarshducoder**
Please review my categories page

live link: https://quizzyfuy.netlify.app/quiz%20html/quiz.html

Following are the features of this page:

- Card: cards are created for displaying the varieties of quiz available.
- footer: contains links to all my social websites.

### CODE REVIEW

File: categories/quiz.css [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+color: blue;
+
+}
+
+.card-heading {
+    margin-left: 0.3rem;
+  }
+  
+  .author {
+    padding: 2px;
```
Comment (i-am-a-codegirl) [inline categories/quiz.css:37]: use rems or ems instead of px for better responsiveness

File: categories/quiz.css [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+      flex-direction: row;
+      gap: 4rem;
+      justify-content: center;
+      align-items: center;
+    
+    /* grid-template-areas: 200px 1fr; */
+    /* align-items: center; 
+    padding: 3.5rem;
+    text-align: center;
+    margin-left: 1rem;
```
Comment (i-am-a-codegirl) [inline categories/quiz.css]: try deleting unwanted code and rest of the code looks fine

File: categories/quiz.css [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+  list-style: none;
+  display: flex;
+  margin: 2rem;
+  gap: 10px;
+  align-items: center;
+  justify-content: center;
+  
+}
+
+@media(max-width: 480px){
```
Comment (i-am-a-codegirl) [inline categories/quiz.css:201]: shift media queries in the end of css

**Review by i-am-a-codegirl — APPROVED** [review]

File: categories/quiz.css [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+    background-color: var(--primary-color);
+}
+h1{
+    color: white;
+    text-align: center;
+}
+
+.imp-text{
+    font-family: 'Redressed', cursive;
+font-family: 'Roboto', sans-serif;
```
Comment (khushijohri2001) [inline categories/quiz.css]: keep the indentation uniform

File: categories/quiz.css [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+  .card-box{
+    display: flex;
+    flex-direction: column;
+    gap: 0.24rem;
+
+  }
+
+.card-one{
+  display: flex;
+
```
Comment (khushijohri2001) [inline categories/quiz.css:64]: remove extra space

**Review by khushijohri2001 — APPROVED** [review]

**Pull request merged**